app/src/main/res/values-de/strings-de.xml

```diff
@@ -289,14 +289,14 @@
 	<string name="copy_all_settings">Alle Einstellungen kopieren</string>
 	<string name="sensor_is_ready_please_enter_double_calibration">Sensor ist bereit, zum Start zweifach kalibrieren.</string>
 	<string name="in_order_to_get_started_please_perform_two_finger">Um die Einrichtung abzuschließen, werden zwei Blutglukosewerte benötigt.</string>
-	<string name="enter_first_bg_value">Erster BG Wert</string>
-	<string name="enter_second_bg_value">Zweiter BG Wert</string>
+	<string name="enter_first_bg_value">Erster BG-Wert</string>
+	<string name="enter_second_bg_value">Zweiter BG-Wert</string>
 	<string name="language_editor">Angezeigte Sprache</string>
 	<string name="configure_missed_readings">Einstellungen für verpasste Werte</string>
 	<string name="install_pebble_classic_trend_watchface">Pebble Classic Trend Watchface installieren</string>
 	<string name="install_pebble_trend_watchface">Pebble Trend Watchface installieren</string>
 	<string name="install_pebble_watchface">Pebble Watchface installieren</string>
-	<string name="glucose_history">Glucose Geschichte</string>
+	<string name="glucose_history">BG-Historie</string>
```

Comment:
The english has "Glucose" not "BG" which I understand would be abbreviation for Blood Glucose which is not the same.